beetmoverscript/utils.py

```diff
@@ -175,16 +175,16 @@ def generate_beetmover_template_args(context):
 def _generate_beetmover_template_args_maven(task, release_props):
     tmpl_args = {
         'artifact_id': task['payload']['artifact_id'],
-        'branch': release_props['branch'],
-        'buildid': release_props['buildid'],
         'product': release_props['appName'],
         'template_key': 'maven_{}'.format(release_props['appName']),
     }
 
     # Change version number to major.minor.buildId because that's what the build task produces
     version = FirefoxVersion.parse(task['payload']['version'])
     tmpl_args['version'] = '{}.{}.{}'.format(
-        version.major_number, version.minor_number, tmpl_args['buildid']
+        version.major_number,
+        version.minor_number,
+        release_props.get('buildid', version.patch_number)
```

Comment:
@JohanLorenzo Do you think this works fine for `geckoview`?

Reply:
It should work with geckoview. However, `version.patch_number` can be `None`. I'd be more strict about this number:

```python
version = [version.major_number, version.minor_number, release_props.get('buildid', version.patch_number)]
if any(number is None for number in version):
    raise TaskVerificationError('At least one digit is undefined. Got: {}'.format(version))
tmpl_args['version'] = '.'.join(version)
```